src/commcare_cloud/ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -18,10 +18,8 @@ http {
         access_log /var/log/{{ nginx_access_log_name }};
         error_log /var/log/{{ nginx_error_log_name }} warn;
 
-        log_format timing '[$time_local] $upstream_cache_status $request $status $request_time $http_referer';
-        log_format rt_cache '$remote_addr - $remote_user - $upstream_cache_status - [$time_local] '
-                    '"$request" $status $request_length $body_bytes_sent '
-                    '"$http_referer" "$http_user_agent"';
+        log_format timing '[$time_local] $upstream_cache_status $request $status $request_time $request_length $body_bytes_sent $http_referer';
```

Comment:
prefer `$bytes_sent` over `$body_bytes_sent` since it includes headers etc:
```suggestion
        log_format timing '[$time_local] $upstream_cache_status $request $status $request_time $request_length $bytes_sent $http_referer';
```